<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageOptionDialog</class>
 <widget class="QDialog" name="ImageOptionDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Use the dark reference to normalize the multispectral image</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="normalize_checkbox">
       <property name="text">
        <string>Normalize</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>The RAYN Vision System Camera optical system is calibrated and it is possible to compensate images taken by this camera for for lens distortion</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>[Experimental] Correct image for light intensity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="done_button">
       <property name="text">
        <string>Done</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="light_correction_checkbox">
       <property name="text">
        <string>Apply Light Intensity Correction for RAYN Vision Systems with 120° lens</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Select Lens Angle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lens_angle_combobox">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>No compensation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>60°</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>120° (legacy)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
